CHANGELOG.md

```diff
@@ -1,6 +1,36 @@
+# dbt_social_media_reporting v0.5.0
+
+## Upstream Breaking Changes (Twitter Organic)
+[PR #12](https://github.com/fivetran/dbt_twitter_organic_source/pull/12) from the upstream `dbt_twitter_organic_source` package includes the following breaking change updates:
+
+- The source defined in the `src_twitter_organic.yml` file has been renamed from `twitter_organic` to `twitter` to align with the default schema name used by the upstream Fivetran connector.
+    - If you're referencing sources from the upstream Twitter Organic packages, please update your source references as needed. See below for the full scope of source changes.
+
+| **New Source Reference** | **Old Source Reference** |
+|----------------------------------|----------------------------------|
+| `"{{ source('twitter','account_history') }}"` | `"{{ source('twitter_organic','account_history') }}"` |
+| `"{{ source('twitter','organic_tweet_report') }}"` | `"{{ source('twitter_organic','organic_tweet_report') }}"` |
+| `"{{ source('twitter','tweet') }}"` | `"{{ source('twitter_organic','tweet') }}"` |
+| `"{{ source('twitter','twitter_user_history') }}"` | `"{{ source('twitter_organic','twitter_user_history') }}"` |
+
+- The default schema name has been modified from `twitter_organic` to now be `twitter` to more closely align with the default schema name generated by the Fivetran connector. Please be aware if you were leveraging the previous default schema then you will need to update the `twitter_organic_schema` variable accordingly. 
+- All identifier variables in the `src_twitter_organic.yml` file have been renamed. If you’re using any of these in your project, please update them accordingly. The changes include: 
+    - Prepending `twitter_organic_*` has been updated to `twitter_*` to align with the schema change.
+    - The spelling of `*_identifer` has been corrected to `*_identifier`.
+
+| **New Identifier Variable Name** | **Old Identifier Variable Name** |
+|----------------------------------|----------------------------------|
+| `twitter_account_history_identifier` | `twitter_organic_account_history_identifer` |
+| `twitter_organic_tweet_report_identifier` | `twitter_organic_organic_tweet_report_identifer` |
+| `twitter_tweet_identifier` | `twitter_organic_tweet_identifer` |
+| `twitter_twitter_user_history_identifier` | `twitter_organic_twitter_user_history_identifer` |
 
 ## Under the Hood
+[PR #11](https://github.com/fivetran/dbt_social_media_reporting/pull/11) includes the following under the hood updates:
+
+- Consistency validation for integration tests has been added for the `social_media_reporting__rollup_report` model. 
+- Updated the maintainer PR, Issue, Feature Request, and Config templates to resemble the most up to date format. 
+- Renamed the Twitter Organic seed files to allow for more testing functionality.
 - Addition of a section tag within the README so the model descriptions may be accessible within the Fivetran UI for Quickstart.
```

Comment:
"Addition of a section tag within the README" I'm not seeing this change in this PR. Was it part of a past PR?

Reply:
It looks like it was part of PR #10. I can add the specific link here for this line.

Reply:
@fivetran-avinash, I just reworked the CHANGELOG to have this section be explicit for the PR reference.